### DISCUSSION TITLE
automated update

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#36] [Ideas]
You should add a way to auto-update the boot usb disk with your release when internet access is working from bleskos...

**Comment by VendelinSlezak** [comment]
Yes, I will surely implement this. One of my goals for BleskOS in 2024 is to make installer to hard disk, so it will be at least in that version.

**Comment by defdefred** [comment]
Will you keep the readonly nature of bleskOS boot disk?

**Comment by VendelinSlezak** [comment]
Update 7 puts BleskOS on small partition on start of USB so you can easily format rest of USB to FAT e.g. in program gparted.

**Comment by defdefred** [comment]
Now update the doc to explain how dd-ing new version without loosing the second partition :-).

**Comment by VendelinSlezak** [comment]
Done :)

**Comment by defdefred** [comment]
crystal clear.

**Comment by defdefred** [comment]
When auto update will be a reality, keeping the last working boot image as a fail safe rescue on the same media could be precious...
The bootloader could propose the new and the failsafe boot image...

**Comment by VendelinSlezak** [comment]
Yes, that's good idea.
